# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dwm/X11R7-lib64.patch
# Copyright (C) 2007 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The libs might be in a lib64 (or so) directory and we have no X11R6 anymore,
(though we would translate that on-the-fly, ...).

 - Rene Rebe <rene@exactcode.de>

--- dwm-5.0/config.mk.vanilla	2008-06-19 10:01:06.000000000 +0200
+++ dwm-5.0/config.mk	2008-06-19 10:05:08.000000000 +0200
@@ -7,16 +7,13 @@
 PREFIX = /usr/local
 MANPREFIX = ${PREFIX}/share/man
 
-X11INC = /usr/X11R6/include
-X11LIB = /usr/X11R6/lib
-
 # Xinerama, comment if you don't want it
-XINERAMALIBS = -L${X11LIB} -lXinerama
+XINERAMALIBS = $(shell pkg-config --libs xinerama)
 XINERAMAFLAGS = -DXINERAMA
 
 # includes and libs
-INCS = -I. -I/usr/include -I${X11INC}
-LIBS = -L/usr/lib -lc -L${X11LIB} -lX11 ${XINERAMALIBS}
+INCS = $(shell pkg-config --cflags x11)
+LIBS = -lc $(shell pkg-config --cflags --libs x11) $(XINERAMALIBS)
 
 # flags
 CPPFLAGS = -DVERSION=\"${VERSION}\" ${XINERAMAFLAGS}
